index.bs

```diff
@@ -697,26 +710,30 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
     1. Assert: |source|'s [=attribution source/source type=] is "`event`".
     1. If the [=list/size=] of the [=attribution report cache=] is greater than or equal to the user
         agent's [=max report cache size=], return.
+    1. Let |fakeConfig| be a new [=event-level trigger configuration=] with the items:
+        : [=event-level trigger configuration/trigger data=]
+        :: |source|'s [=attribution source/attribution mode=][1]
+        : [=event-level trigger configuration/dedup key=]
+        :: null
+        : [=event-level trigger configuration/priority=]
+        :: 0
+        : [=event-level trigger configuration/filters=]
+        :: «[ "`source_type`" → "`event`" ]»
```

Comment:
This doesn't type check, since a the attribution filter map's values must be an ordered set of strings. I think this should be
```
«[ "`source_type`" → « "`event`" » ]»
```

Reply:
Ah, good catch.